deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py

```diff
@@ -0,0 +1,217 @@
+        df.to_csv(csv_filename, index=False)
+
+def run_artillery(args):
+    """Execute Artillery tests.
+    Args:
+        args (ArgumentParser): Script parameters.
+    """
+    json_filename = gen_log_filename(args.logs)
+
+    params = f"-v {gen_artillery_params(args)}"
+    target = f"-t {gen_url(args.ip)}"
+
+    # Enable Quiet Mode (Artillery)
+    quiet = ''
+
+    if not args.debug:
+        quiet = f"-q"
+
+    output = f"-o {json_filename}"
+    script = f'{args.artillery}'
+
+    command = f"artillery run {params} {target} {quiet} {output} {script}"
+
+    run(command, shell=True)
+    convert_json_to_csv(args, json_filename)
+
+def get_script_arguments():
+    parser = argparse.ArgumentParser(usage='%(prog)s [options]', 
+                                     description='Script to Run Dashboard Saturation Tests',
+                                     formatter_class=argparse.RawTextHelpFormatter)
+
+    parser.add_argument('-l', '--logs', dest='logs', default='logs/',
+                        help=f'Directory to store the logs. Default "logs".')
+    
+    parser.add_argument('-s', '--screenshots', dest='screenshots', default='screenshots/',
+                        help=f'Directory to store the screenshots. Default "screenshot".')
+    
+    parser.add_argument('-c', '--csv', dest='csv', default='csv/',
+                        help=f'Directory to store the CSVs. Default "csv".')
+
+    parser.add_argument('-o', '--session', dest='session', default='.auth/',
+                        help=f'Directory to store the Sessions. Default ".auth".')
+
+    parser.add_argument('-a', '--artillery', dest='artillery', type=str, default="artillery.yml",
+                        help=f'Path to the Artillery Script. Default "artillery.yml".')
+
+    parser.add_argument('-u', '--user', dest='user', type=str, default= 'admin',
+                        help=f'Wazuh User for the Dashboard. Default "admin".')
+
+    parser.add_argument('-p', '--password', dest='password', type=str, required=True,
+                        help=f'Wazuh Password for the Dashboard.')
+    
+    parser.add_argument('-q', '--quantity', dest='quantity', type=int, default=1,
+                        help=f'Number of Tests to Run. Default {1}')
+    
+    parser.add_argument('-i', '--ip', dest='ip', type=str, required=True,
+                        help=f'Set the Dashboard IP.')
+
+    parser.add_argument('-t', '--type', dest='type', type=str, nargs='+', action='store',
+                        default=['aggregate', 'intermediate'],
+                        help=f'JSON data to create the CSV.')
+    
+    parser.add_argument('-d', '--debug', dest='debug', action='store_true', required=False,
+                        default=False, help='Enable debug mode')
+
+    return parser.parse_args()
+
+def main():
+    script_args = get_script_arguments()
+
+    process_script_arguments(script_args)
+
+    for n in range(0, script_args.quantity):    
+        run_artillery(script_args)
```

Comment:
Consider adding a configurable timeout between iterations. Additionally, please rename the 'quantity' variable to something more descriptive, as its current name is unclear

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5629/commits/ca020ab5fd076f3ff705605233b10f1ca0921806 and https://github.com/wazuh/wazuh-qa/pull/5629/commits/04ae05abe434b8be4ddae4b72fb582503ab38602